README.md

```diff
@@ -172,3 +172,30 @@ list(address.trades())
 ## Contributing
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
+
```

Comment:
Let's keep the `Contributing` section at the end after the examples

Reply:
done